test/docker/Dockerfile

```diff
@@ -0,0 +1,134 @@
+FROM ubuntu:22.04
+
```

Comment:
```suggestion
# This Docker image is build and push to the repository it takes about 6 min to build
# The reason to pre-build it is because it compiles five libraries and an application everytime we run the test
#  docker build -t ghcr.io/jenkinsci/trilead-ssh2:netopeer-3.5.1 .
```

Reply:
@kuisathaverat  in testcontainers we use docker image with name 'netopeer2'. Will testcontainers pick it up if we use 
netopeer-3.5.1?

Reply:
yes, you have to change to use a Docker image instead of build one

Reply:
@kuisathaverat ok I can change this. And if I want to run this locally?

Reply:
IIRC `ghcr.io` it is public if the repo is public so it should work directly